pkg/plugins/platform/local.go

```diff
@@ -79,19 +80,17 @@ func (l *LocalApp) Describe() (string, error) {
 	}
 
 	var output string
-	app := uitable.New()
-	app.MaxColWidth = maxWidth
-	app.AddRow("NAME", "LOCATION", "TYPE", "URL", "STATUS")
-	app.AddRow(l.GetName(), l.AppRoot(), l.GetType(), l.URL(), "running")
-	output = fmt.Sprint(app)
+	appTable := CreateAppTable()
+	RenderAppRow(appTable, l)
+	output = fmt.Sprint(appTable)
 
 	output = output + "\n\nMySQL Credentials\n-----------------\n"
 	dbTable := uitable.New()
 	dbTable.MaxColWidth = maxWidth
 	dbTable.AddRow("Username:", "root")
 	dbTable.AddRow("Password:", "root")
 	dbTable.AddRow("Database name:", "data")
-	dbTable.AddRow("Connection Info:", l.HostName()+":3306")
+	dbTable.AddRow("Connection Info:", l.HostName()+":"+appports.GetPort("db"))
```

Comment:
Probably should be in a different PR right but since reworking getPort() in this one goes here. I'm glad to see improvements like this, and sometimes it just works best to piggy-back them.